modules/telestion-api/src/main/java/de/wuespace/telestion/api/message/HeaderInformation.java

```diff
@@ -0,0 +1,822 @@
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Integer... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Long} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Long} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Long... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Short} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Short} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Short... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Byte} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Byte} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Byte... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Double} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Double} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Double... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Boolean} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Boolean} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Boolean... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Intermediate step to append a stream of strings to the wrapped {@link MultiMap Vert.x headers}.
+	 */
+	private HeaderInformation add(String key, Stream<String> stream) {
+		var list = stream.toList();
+
+		if (contains(key)) {
+			var existing = getAll(key);
+			logger.warn("The header information object already contains values assigned to that key. " +
```

Comment:
According to AWESA, the `S` ("so we did...") is missing and not implied, here. We should add something along the lines of "Overriding key with new value." to this message for clarity.

Reply:
Fixed in [`8fc8edc` (#484)](https://github.com/wuespace/telestion-core/pull/484/commits/8fc8edcb4d2888945d19b1448d4c280e3df55691)